### ISSUE TITLE
FITCount.ceh.ac.uk count results map addition

### ISSUE THREAD

**Issue by JimChiazzese1** [NERC-CEH/fit-count-app#180]
I had a query from Portugal about the FITcount.ceh.ac.uk results. They are very happy with how it works, but they were wondering if it’s possible to also have a satellite image based map, along with the openstreetmap.
To be honest, they are not really sure why or how it would make it a difference, but some suggestion that satellite map might be a quick way of looking at results from different terrains.
 
Certainly not suggesting we change the mapping, but as many applications have the choice of base map, just wondering if it was already a feature of openstreetmap that could be ‘turned on’ or a case of licensing and overlaying a whole other base map.

**Comment by kazlauskis** [comment]
Yes, we could add another layer. We use Mapbox map satellite services and it wouldn't be hard to add a way to switch to another of [their base layers](https://www.mapbox.com/gallery).
